### DISCUSSION TITLE
What wrong?

### DISCUSSION THREAD

**Discussion by ivose** [gotzmann/comet#20] [General]
I inserted "composer require gotzmann/comet" for installing. I wrote the app.php. I inserted "php app.php start". But  http://localhost:8080 or http://127.0.0.1:8080 simply doesn't work. Please update your comet. And make to usable also in xampp-server.

**Comment by baltasarcs** [comment]
Same problem here. When using the available docker image, the vendor directory is not created in the root folder, consequently the dependencies are not installed and the server is not started. I created another docker image that solves this problem, but even with the server started and using the available examples, nothing is returned.

**Comment by finwe108** [comment]
> I inserted "composer require gotzmann/comet" for installing. I wrote the app.php. I inserted "php app.php start". But http://localhost:8080 or http://127.0.0.1:8080 simply doesn't work. Please update your comet. And make to usable also in xampp-server.

Is there a solution to this already? Not working in xampp for me too
